docs/dm/zh_CN/data-interaction-details.md

```diff
@@ -0,0 +1,40 @@
+数据流过程
```

Comment:
It's awkward to provide this single document, why user need it?  user want to debug every component or task?
so we also need to provide more document about how to debug problem of every component and task. It's indeed!
This is an optimization point